app/routes/rentals/show.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.queryRecord('rental', { title: params.title })
+  },
+
+  setupController(controller, model) {
```

Comment:
@sbatson5 it's slightly controversial, but I personally think it's unnecessary and unexpected, since you expect the `model` property to be set with what's returned from the model hook.